pkg/services/featuremgmt/service.go

```diff
@@ -74,3 +75,12 @@ func ProvideManagerService(cfg *setting.Cfg, licensing models.Licensing) (*Featu
 func ProvideToggles(mgmt *FeatureManager) FeatureToggles {
 	return mgmt
 }
+
+func (fm *FeatureManager) GetUsageStats(ctx context.Context) map[string]int {
+	enabled := fm.GetEnabled(ctx)
+	stats := make(map[string]int, len(enabled))
+	for featureName := range enabled {
+		stats[fmt.Sprintf("stats.features.%s.count", featureName)] = 1
```

Comment:
We don't have a strict naming convention for feature names, e.g.:

- `database_metrics`
- `service-accounts`
- `envelopeEncryption`

I wonder if I should convert them all to snake case or would simply lower casing be enough.

Reply:
Also, some names are pretty long: `azureMonitorResourcePickerForMetrics`, need to figure out the max length for metric name